### PR TITLE
System requirements updated for 3.2 release

### PR DESCRIPTION
**Pull request by michaelstingl**
@fmoc @HanaGemela that's my plan for the customer announcement for the branded 3.2 release 

(based on https://cloud.owncloud.com/index.php/f/5870330)

No Backport

### CODE REVIEW

**Review by mmattel — APPROVED** [review]
Fine to me, another review from @fmoc and @TheOneRing is welcomed

File: modules/ROOT/pages/installing.adoc [hunk 1]
```diff
@@ -30,7 +30,7 @@ Depending on the operating system used, some minimum system requirements need to
 
 Windows::
 * Windows 7 or later
-** x86 with 32-bit or x86-64 with 64-bit
```
Comment (TheOneRing) [inline modules/ROOT/pages/installing.adoc:33]: I think there is still a x64 too much.
Independent of that:
- 32bit isn't dropped, but will be dropped one day.
- I'd remove Windows 7, things might still run but it is 100% untested. Windows 7 is eol.
- In general I'd say we require Windows 10+

Comment (michaelstingl) [reply modules/ROOT/pages/installing.adoc:33]: > * 32bit isn't dropped, but will be dropped one day.

We don't need to drop it, but with 3+ I no longer want to support it.

Comment (michaelstingl) [reply modules/ROOT/pages/installing.adoc:33]: > * I'd remove Windows 7, things might still run but it is 100% untested. Windows 7 is eol.

Works for me. I changed it

Comment (TheOneRing) [reply modules/ROOT/pages/installing.adoc:33]: Windows 8 is also eol :D

File: modules/ROOT/pages/installing.adoc [hunk 2]
```diff
@@ -29,8 +29,8 @@ You can download the latest version of the ownCloud Desktop App from the {deskto
 Depending on the operating system used, some minimum system requirements need to be met. ownCloud provides Linux packages for a variety of Linux distributions, see the list of supported distros below.
 
 Windows::
-* Windows 7 or later
-** x86 with 32-bit or x86-64 with 64-bit
+* Windows 8 or later
+** x86-64 with 64-bit
```
Comment (TheOneRing) [inline modules/ROOT/pages/installing.adoc:33]:
```suggestion
** with 64-bit
```

**Review by HanaGemela — APPROVED** [review]

File: modules/ROOT/pages/installing.adoc [hunk 3]
```diff
@@ -29,8 +29,8 @@ You can download the latest version of the ownCloud Desktop App from the {deskto
 Depending on the operating system used, some minimum system requirements need to be met. ownCloud provides Linux packages for a variety of Linux distributions, see the list of supported distros below.
 
 Windows::
-* Windows 7 or later
-** x86 with 32-bit or x86-64 with 64-bit
+* Windows 8 or later
+** with 64-bit
```
Comment (mmattel) [inline modules/ROOT/pages/installing.adoc:33]:
```suggestion
* Windows 8 or later (x64)
```

**Pull request merged**